sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -610,9 +610,10 @@ def analyze_sentiment(  # type: ignore
         :type documents:
             list[str] or list[~azure.ai.textanalytics.TextDocumentInput] or
             list[dict[str, str]]
-        :keyword bool show_opinion_mining: Whether to mine the opinions of a sentence and conduct more
+        :keyword bool disable_opinion_mining: Whether to disable the opinion mining feature, which
```

Comment:
@abhahn i think we're going to leave it as `show_opinion_mining`, at least for this release

Reply:
Ok, I have reverted it back to `show_opinion_mining`.